gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -107,11 +124,27 @@ func (o *options) validate() error {
 	if _, err := os.Stat(o.flStyles); os.IsNotExist(err) {
 		return fmt.Errorf("styles path '%s' doesn't exists.", o.flStyles)
 	}
+	if o.oauthTokenFile != "" && o.gcsCredentialsFile != "" {
```

Comment:
Why are we building all of this ourselves instead of using `cloud.google.com/go/storage`?

Reply:
Not sure I agree - "rewriting" it to build an authenticated client and use it is simple and we don't maintain any code that way. Let's take that approach.

Reply:
Updated